packages/span-transformations/src/jaeger.ts

```diff
@@ -0,0 +1,70 @@
+import { SpanAttributes, SpanKind } from '@opentelemetry/api';
+import { timeInputToHrTime } from '@opentelemetry/core';
+import { ReadableSpan } from '@opentelemetry/sdk-trace-base';
+
+const getJaegerValueForTag = (jaegerTagKey, tags) => tags.filter(({ key }) => jaegerTagKey === key)?.[0]?.value;
+const convertJaegerTagsToAttributes = (tags): SpanAttributes => {
+    const spanAttributes: SpanAttributes = {};
+    tags.forEach(({ key, value }) => {
+        spanAttributes[key] = value;
+    });
+    return spanAttributes;
+};
+
+const getOtelKindFromJaegerKind = (jaegerKind: string) => {
+    switch (jaegerKind) {
+        case 'client':
+            return SpanKind.CLIENT;
+        case 'producer':
+            return SpanKind.PRODUCER;
+        case 'server':
+            return SpanKind.SERVER;
+        case 'consumer':
+            return SpanKind.CONSUMER;
+        default:
+            return SpanKind.INTERNAL;
+    }
+};
+
+const getParentSpanID = (jaegerSpan) => {
+    let parentSpanId = '';
+    try {
+        parentSpanId = jaegerSpan.references?.filter(({ refType }) => refType === 'CHILD_OF')[0].spanID;
+    } catch (ex) {
+        console.error('Failed to get parent span ID', ex);
+    }
+
+    return parentSpanId;
+};
+
+export const convertJaegerSpanToOtelReadableSpan = (jaegerSpan): ReadableSpan => {
+    const durationMillis = jaegerSpan.duration / 1000;
+    const startDateMillis = jaegerSpan.startTime / 1000;
+    const endDateMillis = timeInputToHrTime(new Date(startDateMillis + durationMillis));
+
+    return {
+        name: jaegerSpan.operationName,
+        kind: getOtelKindFromJaegerKind(getJaegerValueForTag('span.kind', jaegerSpan.tags)),
+        attributes: convertJaegerTagsToAttributes(jaegerSpan.tags),
```

Comment:
Should we filter out tags that hold data that is not an otel attribute? such as 'span.kind', 'otel.library.name' etc

Reply:
I guess we should. but I am not sure we want to do this right now. @habmic thoughts? maybe we leave as is now and add an issue in the repo about this ?

Reply:
I agree @aspectom, lets create an issue

Reply:
Done